docs/user-guide/config-examples.md

```diff
@@ -17,6 +17,7 @@ all of the nixvim options are accessible no matter how you are using it (flake,
 | [GaetanLepage/nix-config] | Home-manager |
 | [gwg313/nvim-nix] | |
 | [hbjydev/hvim] | |
+| [MikaelFangel/nixvim-config] | |
```

Comment:
The repo description is:

> An easy-setup configuration for NixVim, focused on straightforward customization. 

Perhaps we could quote or paraphrase that in the description? Or would you prefer to leave that column for more significant/important notes?

Reply:
It's good to have it. Done